content/other/qml-benchmarks/binarized-mnist/using_this_dataset.md

```diff
@@ -0,0 +1 @@
+# Using this Dataset
```

Comment:
@josephbowles 
This is where to add text for the "Using this dataset" section.

An example of how we did this for the other qml benchmarks is [here](https://github.com/XanaduAI/pennylane-datasets/blob/main/content/other/qml-benchmarks/hyperplanes/using_this_dataset.md).